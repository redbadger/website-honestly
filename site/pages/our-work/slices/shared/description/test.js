// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Description from '.';

describe('site/pages/our-work/slices/shared/description', () => {
  it('renders the correct class by default', () => {
    const component = shallow(<Description>Some text</Description>);
    expect(component).toMatchSnapshot();
  });

  it('renders the maxWidth class', () => {
    const component = shallow(<Description maxWidth>Some text that needs to wrap</Description>);
    expect(component).toMatchSnapshot();
  });
});
