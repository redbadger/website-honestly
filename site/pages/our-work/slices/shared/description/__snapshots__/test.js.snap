// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/slices/shared/description renders the correct class by default 1`] = `
<p
  className="description"
>
  Some text
</p>
`;

exports[`site/pages/our-work/slices/shared/description renders the maxWidth class 1`] = `
<p
  className="descriptionMaxWidth"
>
  Some text that needs to wrap
</p>
`;
